db/sqldb/portfolio.go

```diff
@@ -3,45 +3,50 @@ package sqldb
 import (
 	"errors"
 	"fmt"
+	"time"
 
 	"github.com/RMI/pacta/db"
 	"github.com/RMI/pacta/pacta"
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgtype"
 )
 
-const portfolioSelectColumns = `
-	portfolio.id,
-	portfolio.owner_id,
-	portfolio.name,
-	portfolio.description,
-	portfolio.created_at,
-	portfolio.holdings_date,
-	portfolio.blob_id,
-	portfolio.admin_debug_enabled,
-	portfolio.number_of_rows 
+const portfolioQueryStanza = `
+	SELECT
+		portfolio.id,
+		portfolio.owner_id,
+		portfolio.name,
+		portfolio.description,
+		portfolio.created_at,
+		portfolio.holdings_date,
+		portfolio.blob_id,
+		portfolio.admin_debug_enabled,
+		portfolio.number_of_rows,
+		portfolio_group_membership.portfolio_group_id,
+		portfolio_group_membership.created_at
+	FROM portfolio
+	LEFT JOIN portfolio_group_membership 
+	ON portfolio_group_membership.portfolio_id = portfolio.id
```

Comment:
Noted below, but the downside of this approach is that we repeatedly load all the portfolio information, once for each group membership. As long as the portfolio metadata is a small known quantity (e.g. no giant blob columns), and we don't expect the cardinality of groups per portfolio to be in the thousands, this is likely fine, but otherwise I'd just batch load membership info in a separate query given all the portfolio IDs.

Same deal for `PortfolioGroup`s

Reply:
I don't think that's the case - the `portfolio_group_membership` table is just the join table, not the raw `portfolio` table, so even if we stored a blob on the `portfolio` table, this wouldn't pull that in.

Reply:
An example to better illustrate my point:

```sql
CREATE TABLE portfolio (
	id TEXT NOT NULL PRIMARY KEY,
	some_giant_blob TEXT NOT NULL
);

CREATE TABLE some_join_table (
	id TEXT NOT NULL PRIMARY KEY,
	portfolio_id TEXT NOT NULL REFERENCES portfolio (id)
);

INSERT INTO portfolio (id, some_giant_blob) VALUES ('p1', 'imagine this is huge');
INSERT INTO some_join_table (id, portfolio_id) VALUES ('sjt1', 'p1'), ('sjt2', 'p1'), ('sjt3', 'p1');

SELECT
	portfolio.id,
	portfolio.some_giant_blob,
	some_join_table.id
FROM portfolio
LEFT JOIN some_join_table
	ON some_join_table.portfolio_id = portfolio.id;

 id |   some_giant_blob    |  id
----+----------------------+------
 p1 | imagine this is huge | sjt1
 p1 | imagine this is huge | sjt2
 p1 | imagine this is huge | sjt3
(3 rows)
```

See how we load the portfolio once for each relationship in the other table? That's potentially a lot of extra data to transmit and parse if the portfolio is large or there are a lot of groups.

Reply:
Sorry I misunderstood this - I see the point and I know a fix! I'll pursue this afternoon.